db/blob_log_format.cc

```diff
@@ -0,0 +1,245 @@
+#include "db/blob_log_format.h"
+#include "util/coding.h"
+#include "util/crc32c.h"
+
+
+namespace rocksdb {
+namespace blob_log {
+
+
+BlobLogHeader::BlobLogHeader()
+: magic_number_(kMagicNumber),
```

Comment:
I suggest we keep a version number in the header. We can't always rely on magic number to identify versions.